backend/common/src/main/java/ai/verta/modeldb/common/CommonDBUtil.java

```diff
@@ -313,12 +313,6 @@ protected void runLiquibaseMigration(
     // Change liquibase default table names
     String changeLogTableName = "database_change_log";
     String changeLogLockTableName = "database_change_log_lock";
-    if (config.getRdbConfiguration().isH2()) {
-      // H2 upper cases all table names, and liquibase has issues if you don't make this also upper
-      // case.
-      changeLogTableName = changeLogTableName.toUpperCase();
-      changeLogLockTableName = changeLogLockTableName.toUpperCase();
-    }
```

Comment:
has this changed? When I originally put this code in when writing tests against H2 for UAC, this was necessary. What has changed to make it not needed any more?

Reply:
https://github.com/VertaAI/modeldb/pull/3228#discussion_r1006110012